scripts/forc-update.ts

```diff
@@ -2,6 +2,7 @@
 
 import sh from 'shelljs';
 
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Same here; it seems to me that these methods using `shelljs` are synchronous, discarding the need for the `async` directive.

Reply:
Yes, we are right. I am going to just remove the `async` here